appservice/src/createAppService/AppKind.ts

```diff
@@ -10,7 +10,8 @@ export enum WebsiteOS {
 
 export enum AppKind {
     app = 'app',
-    functionapp = 'functionapp'
+    functionapp = 'functionapp',
+    workflowapp = 'workflowapp'
```

Comment:
These strings don't match here: https://github.com/microsoft/vscode-azuretools/pull/886/files#diff-286a2b3db8a91df266b072104bec32dca8d8124f038dcc0df9beea633788351dR14

Reply:
I have provided a comment in the other PR to just keep one value, its because kind usually is a combination of strings for workflowapp and we should not provide a strict order else string matching will not be correct.
today siteKind if it contains workflowApp it means its a logic apps in preview.